### PR TITLE
Observability architecture

### LINKED ISSUE #321: Create architecture diagrams and docs for ACM Observability on new Observability cluster

### PR DESCRIPTION
**Pull request by jbasu01**
Created the NERC observability architecture along with documentation that details the steps to enable the observability service in RHACM and the observability workflow.  The documentation also includes the technologies utilized to accomplish providing visibility to all the cluster metrics.

fixes https://github.com/nerc-project/operations/issues/321

### CODE REVIEW

File: architecture/observability/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+![plot](./img/Observability-Architecture.png)
+
+# Technology
+
+The following technologies have been chosen for this solution:
+
+**Red Hat Advanced Cluster Management (RHACM):** This component helps overcome the challenges of managing multi-clusters environments and ensures appropriate enforcement of configurations and policies across the clusters for governance and compliance.
+
+**Red Hat Advanced Cluster Management Observability Service:** The observability service provides a centralized hub for metrics, alerting, and monitoring of platforms for a multi-cluster environment. In addition, the observability component also focuses on displaying cluster health metrics, which describes the control plane health, cluster optimization and resource utilization. gets deployed automatically to each cluster when the service is enabled.
```
Comment (computate) [inline architecture/observability/README.md]: @jbasu01 My only feedback is this sentance could be capitalized "gets deployed automatically... " This documentation nicely describes all the components of metrics and alerts. Thanks very much for putting this together!

**Review by computate — CHANGES_REQUESTED** [review]

File: architecture/observability/README.md (hunk 1 above)
Comment (jbasu01) [reply architecture/observability/README.md]: Good catch. Actually, that sentence does fit there. I'll make the appropriate changes.

**Comment by schwesig** [comment]
supporting info:
[How Your Grafana Can Fetch Metrics From Red Hat Advanced Cluster Management Observability (Observatorium and Thanos)](https://www.redhat.com/en/blog/how-your-grafana-can-fetch-metrics-from-red-hat-advanced-cluster-management-observability-observatorium-and-thanos)

**Review by computate — APPROVED** [review]

**Review by schwesig — APPROVED** [review]
great work, as discussed in our meetings yesterday, all approved
thanks @jbasu01 & @harshil-codes

**Pull request merged**